app/views/pages/get-started/contact-us-tutorial/setting-up-endpoint.liquid

```diff
@@ -0,0 +1,351 @@
+
+{% include 'alert/note', content: 'All parameters submitted by the user are available in `context.params`. This allows you to access and use these parameters within your Liquid templates, making your applications dynamic and responsive to user input. Read more about [context.params](/api-reference/liquid/platformos-objects#context-params) in the platformOS documentation.' %}
+
+## Adding parameters to your template
+
+The following sections are not part of the main task; they are provided for demonstration purposes. However, we recommend following along and using the code snippets to gain a better understanding of this concept. If you understand how context.params work already, skip this chapter and continue from [Using context.params](/get-started/contact-us-tutorial/understanding-context-params#using-context-params-in-the-create-liquid-file).
+
+First, let's see how parameters can be accessed in a Liquid template. In the previous step, we added 
+
+{% raw %}
+```liquid
+Hello {{ context.params }}
+```
+{% endraw %}
+
+to our `/app/views/pages/contacts/create.liquid` file.
+
+For testing purposes, temporarily add the following code to your `app/views/pages/index.html.liquid` file:
+
+#### app/views/pages/index.html.liquid
+
+{% raw %}
+```liquid
+My parameters {{ context.params }}
+
+<h2>Contact Us</h2>
+<form action="/contacts/create" method="post">
+<input type="hidden" name="authenticity_token" value="{{ context.authenticity_token }}">
+    <div>
+        <label for="email">Email</label>
+        <input type="text" name="contact[email]" id="email">
+    </div>
+    <div>
+      <textarea name="contact[body]"></textarea>
+    </div>
+    <input type="submit" value="Send">
+  </form>
+```
+{% endraw %}
+
+Navigate to your home page. You should see something like:
+
+{% # TODO: check size %}
+<img loading="lazy" class="w-full" src="{{ 'images/get-started/contact-us-tutorial/contact_us_parameters.png' | asset_url }}" alt="Display of parameters on the homepage">
+
+
+```html
+My parameters {"format":"html"}
+```
+
+You might notice that there are no parameters initially displayed, except the default HTML format. This is because parameters need to be explicitly provided in the URL or through form submissions.
+
+## Example URL with parameters
+
+To see how parameters work, add a query parameter to the URL in your browser:
+
+```html
+/?hello_world=hi
+```
+
+The page will display:
+
+```html
+My parameters: {"hello_world":"hi","format":"html"}
+```
+
+Here, `hello_world` is the key, and `hi` is its value.
+
+{% # TODO: check size %}
+<img loading="lazy" class="w-full" src="{{ 'images/get-started/contact-us-tutorial/parameters_hello_hi.png' | asset_url }}" alt="Screenshot showing key 'hello_world' with value 'hi'">
+
+## Providing multiple parameters
```

Comment:
well we want this article to include all the basic information that is required to be able to work with platformOS - and how people can access query params in platformOS feels like one of the most important things to know, so I believe it would be great to have it here - so I guess it might be the matter of wording to clarify our intention here?

Reply:
Added some extra wording at the beginning of the chapter.